packages/trader/src/Stores/Modules/Portfolio/Helpers/format-response.js

```diff
@@ -2,6 +2,9 @@ import { getUnsupportedContracts } from 'Constants';
 import { getSymbolDisplayName } from 'Stores/Modules/Trading/Helpers/active-symbols';
 import { getMarketInformation } from 'Modules/Reports/Helpers/market-underlying';
 
+const isUnSupportedContract = portfolio_pos => !!getUnsupportedContracts()[portfolio_pos.contract_type] || // check unsupported contract type
+        portfolio_pos.date_start !== portfolio_pos.purchase_time; // for forward start contracts
```

Comment:
Why not just use `is_forward_starting` property from `proposal_open_contract` response?